evennia/scripts/manager.py

```diff
@@ -85,7 +85,7 @@ def get_all_scripts(self, key=None):
             script = []
             dbref = self.dbref(key)
             if dbref or dbref == 0:
-                script = [self.dbref_search(dbref)]
+                script = filter(None, [self.dbref_search(dbref)])
```

Comment:
`filter` is a builtin function in Python:

    filter(...)
    filter(function or None, sequence) -> list, tuple, or string
    
    Return those items of sequence for which function(item) is true.  If
    function is None, return the items that are true.  If sequence is a tuple
    or string, return the same type, else return a list.

So there are no imports needed for this.

For what it's worth, Griatch changed this to a list comprehension instead of using filter for compatibility with Python3 (filter there returns an iterator and not a list).